### ISSUE TITLE
dmd fails to build: 

### ISSUE THREAD

**Issue by trofi** [NixOS/nixpkgs#138376]
`dmd` fails the build as: https://hydra.nixos.org/build/153068055

```
searching for dependencies of /nix/store/rciyy42cshid702l6bryljd4aac14drn-dmd-bootstrap-2.090.1/lib32/libphobos2.so.0.90.1
  ld-linux.so.2 -> not found!
autoPatchelfHook could not satisfy dependency ld-linux.so.2 wanted by /nix/store/rciyy42cshid702l6bryljd4aac14drn-dmd-bootstrap-2.090.1/lib32/libphobos2.so.0.90.1
Add the missing dependencies to the build inputs or set autoPatchelfIgnoreMissingDeps=true
```

### Notify maintainers

@ThomasMader @lionello 

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: dmd
```

**Comment by impl** [comment]
FYI, this should be fixed by #137886.

**Comment by trofi** [comment]
Works in `staging`. Closing.

Thank you!
